metrics/density/README.md

```diff
@@ -0,0 +1,46 @@
+# Kata Containers density metrics tests
+
+This directory contains a number of tests to help measure container
+density. Some measures are based around the PSS of the runtime components,
```

Comment:
I'd consider defining the term container density. Something like:

> to help measure the number of containers that can be run simultaneously, known as *container density*.

Also how about a definition of PSS?

In fact, it might even be worth creating a doc specifically for abbreviations and acronymns like this that all test docs could refer to?